vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -90,13 +90,25 @@
       <AdditionalDependencies>Shlwapi.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <AdditionalOptions>-minpdbpathlen:256</AdditionalOptions>
     </Link>
+    <Midl>
+      <AdditionalIncludeDirectories>$(ReactNativeWindowsDir)\Microsoft.ReactNative;$(ReactNativeWindowsDir)\Microsoft.ReactNative.Cxx;</AdditionalIncludeDirectories>
```

Comment:
>$(ReactNativeWindowsDir)\Microsoft.ReactNative.Cxx [](start = 83, length = 50)

I wonder why is it need for the MIDL directories.
It would be better to have this part in the Microsoft.ReactNative.Cxx shared project instead.

Reply:
It's for 
      #include "NamespaceRedirect.h"
in Microsoft.ReactNative\IReactContext.idl

I think it's a backwards build dependency we should change.

---
In reply to: [423288009](https://github.com/microsoft/react-native-windows/pull/4848#discussion_r423288009) [](ancestors = 423288009)